### ISSUE TITLE
Support GHC 7.10

### ISSUE THREAD

**Issue by FranklinChen** [music-suite/music-parts#19]
GHC 7.10 compile error:

```console
src/Music/Parts.hs:181:13:
    Non type-variable argument
      in the constraint: Data.String.IsString [a]
    (Use FlexibleContexts to permit this)
    When checking that ‘addS’ has the inferred type
      addS :: forall a. (Eq a, Data.String.IsString [a]) => [a] -> [a]
    In an equation for ‘show’:
        show (Part Solo instr subp)
          = "Solo " ++ show instr ++ addS (show subp)
          where
              addS "" = ""
              addS x = " " ++ x
    In the instance declaration for ‘Show Part’
```

**Comment by hanshoglund** [comment]
I have no oppertunity to test GHC 7.10 at the moment. You can try 

- `cabal unpack` and cd into source coude
- Add `FlexibleContexts` at the concerned file (as per the error message)
- `cabal install`

Patches and PRs welcome.

**Comment by hanshoglund** [comment]
Assuming resolved by #20, closing.
